package-lock.json

```diff
@@ -4763,6 +4720,15 @@
         "node": ">=12.0.0"
       }
     },
+    "node_modules/cypress-real-events": {
+      "version": "1.7.0",
+      "resolved": "https://cognigy.pkgs.visualstudio.com/_packaging/cognigy-feed/npm/registry/cypress-real-events/-/cypress-real-events-1.7.0.tgz",
```

Comment:
Please reinstall this without your `.npmrc` redirecting it to a private repo

Reply:
Fixed. thanks!